_includes/templates/download.html

```diff
@@ -69,7 +69,7 @@
       <a href="{{ PATH_PREFIX }}/SHA256SUMS.asc" class="dl">{{ page.downloadsig }}</a><br>
       <a href="{{ PATH_PREFIX }}/{{ FILE_PREFIX }}.torrent" class="dl">{{ page.downloadtorrent }}</a>
         {% if magnet %} <a href="{{ magnet | replace: '&', '\&amp;'}}" class="magnetlink" data-proofer-ignore></a>{% endif %}<br>
-      <a href="https://github.com/bitcoin/bitcoin" class="dl">{{ page.source }}</a><br>
+      <a href="https://github.com/bitcoin/bitcoin/releases/tag/v{{CURRENT_RELEASE}}" class="dl">{{ page.source }}</a><br>
```

Comment:
The archive generated by GitHub might not be deterministically created. I'd prefer to link to the archive in our /bin/ folder. (And then optionally to the general Bitcoin Core GitHub, as well)